pom.xml

```diff
@@ -411,6 +411,11 @@ limitations under the License.
         <artifactId>hsqldb</artifactId>
         <version>${hsqldb.version}</version>
       </dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
```

Comment:
Would you please use spaces instead of tabs for indentation like used in the file?

Reply:
done